scripts/shutdownLocalCluster.sh

```diff
@@ -1,21 +1,37 @@
 #!/bin/bash
+params=("$@")
 
-. `dirname $0`/cluster-run-common.sh
+lib="$(dirname $(readlink -f ${BASH_SOURCE[0]}))/cluster-run-common.sh"
+if [[ -f "$lib" ]]; then
+    . "$lib"
+else
+    echo "could not source $lib"
+    exit 1
+fi
+
+if [[ -f cluster/startup_parameters ]];then
+    if [[ -z "${params[@]}" ]]; then
+        string="$(< cluster/startup_parameters)"
+        params=( $string )
+    fi
+fi
+
+parse_args "${params[@]}"
```

Comment:
This is bash magic, does this call parse_args with a single argument or with a list consisting of the entries in the params array? I can never remember.

Reply:
This creates multiple arguments that can potentially contain white spaces.

Practically for the script in its entirety spaces in arguments are not supported. This is the case because we do not store the arguments on single lines, what would be required for whitespace support.

Reply:
ok